ownCloud/Client/Actions/Action.swift

```diff
@@ -281,4 +321,152 @@ class Action : NSObject {
 	var position : ActionPosition {
 		return type(of: self).applicablePosition(forContext: context)
 	}
+
+	// MARK: - Sharing
```

Comment:
Please put the Sharing methods into an extension to the `Action` class:
```suggestion
}

// MARK: - Sharing
extension Action {
```